sdk/websites/Azure.ResourceManager.AppService/src/Generated/WebSiteCollection.cs

```diff
@@ -276,7 +276,7 @@ public async virtual Task<Response<bool>> ExistsAsync(string name, CancellationT
                 throw new ArgumentNullException(nameof(name));
             }
 
-            using var scope = _clientDiagnostics.CreateScope("WebSiteCollection.ExistsAsync");
+            using var scope = _clientDiagnostics.CreateScope("WebSiteCollection.Exists");
```

Comment:
Why change this from Async to sync?

Reply:
It's a minor fix to keep the scope name consistent, in other methods like [`CreateOrUpdateAsync`](https://github.com/Azure/azure-sdk-for-net/blob/d9227a57be17cde5697c6d48dca986d23b347c70/sdk/compute/Azure.ResourceManager.Compute/src/Generated/AvailabilitySetCollection.cs#L106) we always use the name without 'Async" suffix.